playbooks/templates/k8s/namespace/04_portal.yml.j2

```diff
@@ -189,7 +189,7 @@ data:
     ELASTICSEARCH_HOST={{ wai_load_balancer[item].elastic[0] }}
     ELASTICSEARCH_PORT=9200
     ELASTICSEARCH_INDEX_PREFIX=wai
-    ELASTICSEARCH_INDEX_NAME=application-log
+    ELASTICSEARCH_INDEX_NAME=wai-application-log-{{ item }}
```

Comment:
@Valair & @pdavide 
Quindi l'indice è wai-wai-application-log-[wai-prod|wai-stag|way-play] ? Non ci sono troppi wai ?
Nel senso se ELASTICSEARCH_INDEX_PREFIX è un prefix a che serve mettere wai nell'index name ?

Reply:
`ELASTICSEARCH_INDEX_PREFIX` è usato dallo script di build per costruire `ELASTICSEARCH_INDEX_NAME` (e l'index pattern di kibana).
Nel `.env`, l'`ELASTICSEARCH_INDEX_NAME` atteso è quello già completo, es. `wai-application-log*`.
Errore mio, pensavo `item` fosse l'env quindi un risultato del tipo `wai-application-log-production|staging|public-playground` invece, correggimi se sbaglio, dovrebbe essere `ELASTICSEARCH_INDEX_NAME=wai-application-log-{{ wai_portal_environments[item] }}`

Reply:
@Valair 

> @Valair & @pdavide
> Quindi l'indice è wai-wai-application-log-[wai-prod|wai-stag|way-play] ? Non ci sono troppi wai ?
> Nel senso se ELASTICSEARCH_INDEX_PREFIX è un prefix a che serve mettere wai nell'index name ?

@Valair corretto. Ma, domani  me lo spiegherai, non ho capito a che serve ELASTICSEARCH_INDEX_PREFIX

Reply:
@giafar a livello di portale, in effetti mi sembra di vedere che non serva a nulla, potrebbe essere eliminato (verifico meglio e magari faccio una PR post go-live).
Serve sicuramente allo script di build per lo sviluppo locale.

Reply:
- [costruzione dell'index name](https://github.com/AgID/wai-portal/blob/a007a812c486493291913480bc566a8c20d6adcf/build.xml#L64)
- [costruzione dell'index pattern kibana](https://github.com/AgID/wai-portal/blob/a007a812c486493291913480bc566a8c20d6adcf/containers/kibana/index_pattern.json#L3)

Reply:
@Valair quindi per chiarezza potremmo togliere `ELASTICSEARCH_INDEX_PREFIX` dalla configmap e anche da `env-laravel.template`?